app/src/main/java/com/bc/gordiansigner/ui/account/AccountsActivity.kt

```diff
@@ -209,6 +264,25 @@ class AccountsActivity : BaseAppCompatActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+
+        if (resultCode == Activity.RESULT_OK) {
+            when (requestCode) {
+                REQUEST_CODE_INPUT_KEY -> {
```

Comment:
Consider to handle `REQUEST_CODE_INPUT_KEY`. You can take a look on something similar that already implemented. Notice that it does not returns `Activity.RESULT_OK`, so weird :D